core/src/main/res/values/strings.xml

```diff
@@ -187,7 +187,7 @@
         <item quantity="one">%d subscription updated.</item>
         <item quantity="other">%d subscriptions updated.</item>
     </plurals>
-    <string name="add_to_folder">Add to folder</string>
+    <string name="add_tag">Edit tags</string>
```

Comment:
Either the string name or the text should be changed here, so that they say the same thing

Reply:
I think that 'Edit tags' (the text) should prevail here - as tags could also be removed by the user.

Reply:
@keunes Yeah, that is right. I will change the name to edit_tags.